src/amp.js

```diff
@@ -65,6 +65,7 @@ try {
       makeBodyVisible(document, waitForExtensions(window));
     } catch (e) {
       makeBodyVisible(document);
+      throw e;
     } finally {
       perf.tick('e_is');
```

Comment:
I never investigated this originally, but will `perf.tick` itself throw an error if something went wrong with the above `try` block?